src/jit/lclvars.cpp

```diff
@@ -6533,9 +6533,15 @@ void Compiler::lvaDumpRegLocation(unsigned lclNum)
 #ifdef _TARGET_ARM_
     else if (varDsc->TypeGet() == TYP_DOUBLE)
     {
+#ifdef LEGACY_BACKEND
+        // The assigned registers are `lvRegNum:lvOtherReg`
         printf("%3s:%-3s    ", getRegName(varDsc->lvRegNum), getRegName(varDsc->lvOtherReg));
```

Comment:
hmmmm.. interesting, and strange.

I agree that "f4:f5" style does emphasize what is going on, so seems reasonable.